packages/xml-views-validation/test/validators/element/unknown-tag-name-spec.ts

```diff
@@ -0,0 +1,525 @@
+import { partial } from "lodash";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { generateModel } from "@ui5-language-assistant/test-utils";
+import {
+  assertNoIssues as assertNoIssuesBase,
+  assertSingleIssue as assertSingleIssueBase,
+} from "../../test-utils";
+import { validateUnknownTagName } from "../../../src/validators/elements/unknown-tag-name";
+import {
+  getMessage,
+  UNKNOWN_CLASS_IN_NS,
+  UNKNOWN_CLASS_WITHOUT_NS,
+  UNKNOWN_TAG_NAME_IN_CLASS,
+  UNKNOWN_AGGREGATION_IN_CLASS,
+} from "../../../src/utils/messages";
+
+describe("the unknown tag name validation", () => {
+  let ui5SemanticModel: UI5SemanticModel;
+
+  before(async () => {
+    ui5SemanticModel = await generateModel({ version: "1.74.0" });
+  });
+
+  context("true positive scenarios", () => {
+    let assertSingleIssue: (xmlSnippet: string, message: string) => void;
+    before(() => {
+      assertSingleIssue = partial(
```

Comment:
this "before" does not actually call the productive code or take much time.
So while it does not hurt to have it in the before clause, it also does not help much.

Reply:
It's in a `before` because the model is built in a `before` so I couldn't call `partial` with it directly inside the `context`